examples/cockroachdb/README.md

```diff
@@ -0,0 +1,19 @@
+# CockroachDB on Kubernetes as a PetSet
+
+This example deploys CockroachDB on Kubernetes as a PetSet. Note that
+CockroachDB already does replication and thus should not be deployed on
+a persistent volume which already replicates internally. Best performance
+would dictate use of local storage, which is currently not an option (at
+least not as a PetSet).
+
+## Running on minikube
+
+Follow the steps in [minikube.sh](minikube.sh) (or simply run that file).
+
+## Simulating failures
+
+When all (or enough) nodes are up, simulate a failure like this:
+
+```shell
```

Comment:
please provide an example of how to set/get a key:value. An example displaying the replication properties would be nice too (i.e set in 1 read from 5), as would a scale up/down example (kubectl edit the petset and modify replicas to 6, does the new replica catch up to the cluster?). Is there a "master" we should avoid killing during upgrade? 

Optionally, you could write something, bounce the whole cluster (delete pods --all), read the thing. 
Also mentioning read/write throughput and how that scales with replicas will help.